opentdf-saas-okta.yaml

```diff
@@ -0,0 +1,168 @@
+logger:
+  level: debug
+  type: text
+  output: stdout
+# DB and Server configurations are defaulted for local development
+# db:
+#   host: localhost
+#   port: 5432
+#   user: postgres
+#   password: changeme
+#   sslmode: prefer
+#   connect_timeout_seconds: 15
+#   pool:
+#     max_connection_count: 4
+#     min_connection_count: 0
+#     min_idle_connections_count: 0
+#     max_connection_lifetime_seconds: 3600
+#     max_connection_idle_seconds: 1800
+#     health_check_period_seconds: 60
+services:
+  kas:
+    preview:
+      ec_tdf_enabled: false
+      key_management: false
+    root_key: a8c4824daafcfa38ed0d13002e92b08720e6c4fcee67d52e954c1a6e045907d1 # For local development testing only
+    keyring:
+      - kid: e1
+        alg: ec:secp256r1
+      - kid: e1
+        alg: ec:secp256r1
+        legacy: true
+      - kid: r1
+        alg: rsa:2048
+      - kid: r1
+        alg: rsa:2048
+        legacy: true
+  entityresolution:
+    log_level: info
+    mode: virtrusaas
+    # url: http://localhost:8888/auth
+    # clientid: "tdf-entity-resolution"
+    # clientsecret: "secret"
+    # realm: "opentdf"
+    # legacykeycloak: true
+    # inferid:
+    #   from:
+    #     email: true
+    #     username: true
+  # policy is enabled by default in mode 'all'
+  # policy:
+  #   enabled: true
+  #   list_request_limit_default: 1000
+  #   list_request_limit_max: 2500
+  # authorization:
+  #   entitlement_policy_cache:
+  #     enabled: false
+  #     refresh_interval: 30s
+server:
+  public_hostname: localhost
+  tls:
+    enabled: false
+    cert: ./keys/platform.crt
+    key: ./keys/platform-key.pem
+  auth:
+    enabled: true
+    enforceDPoP: false
+    audience: "https://api-develop01.develop.virtru.com"
```

Comment:
- `issuer` updated to point to Virtru Okta OIDC develop platform
- `audience` updated to match audience returned in token